transmit-lib/src/cmr/transmit/urs.clj

```diff
@@ -70,8 +69,9 @@
 (defn get-launchpad-user
   "Returns URS user info associated with a launchpad token"
   [context token]
-  (let [{:keys [status body]} (request-with-auth context {:url-fn #(launchpad-validation-url % token)
-                                                          :method :post :raw? true})]
+  (let [{:keys [status body]} (request-with-auth context {:url-fn #(launchpad-validation-url %)
+                                                          :method :post :raw? true 
+                                                          :http-options {:body (json/generate-string {:token token})}})]
```

Comment:
the documentation for the new post body doesn't indicate that it excepts json.  Have you tested this?

Reply:
This has been fixed to use form urlencoded params instead of json.